website/docs/getting-started/setup/installation-guides/azure-aci.md

```diff
@@ -0,0 +1,69 @@
+---
+description: Deploy Appsmith on Azure Container Instance Service
+sidebar_position: 4
+---
+
+# Azure Container Instance
+Azure Container Instances (ACI) is a fast and simple way to run containers in the cloud.
+In this document, we will guide you through the process of launching an ACI instance and running an Appsmith container on it. The data will be persisted to an Azure Storage Account File Share 
+note: Azure only supports CIFS file shares and NFS file shares are not yet supported.
+
+### Prequisite
+Before you can launch an ACI instance, you need to have an Azure subscription and have the Azure CLI installed on your machine. If you do not have an Azure subscription, you can sign up for a free trial at **[https://azure.com/free](https://azure.com/free)**.
```

Comment:
Can we list the pre-requisites in bulleted format and in the order they need to be installed assuming the user maybe doing it for the first time? 

So something like

Before launching an ACI instance, you need to ensure that the following prerequisites are met.

- [Azure subscription](https://azure.com/free)
- Azure CLI 
- [Create an account on AWS]((https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/)) if you don't already have one.